src/react-components/info-dialog.js

```diff
@@ -174,12 +175,12 @@ class InfoDialog extends Component {
             <p />
             You can file a{" "}
             <a href="https://github.com/mozilla/hubs/issues" target="_blank" rel="noopener noreferrer">
-              Github Issue
+              GitHub Issue
             </a>{" "}
             or e-mail us for support at <a href="mailto:hubs@mozilla.com">hubs@mozilla.com</a>.
             <p />
-            You can also find us in #social on the{" "}
-            <a href="http://webvr-slack.herokuapp.com/" target="_blank" rel="noopener noreferrer">
+            You can also find us in <a href="https://webvr.slack.com/messages/social">#social</a> on the{" "}
+            <a href="https://webvr-slack.herokuapp.com/" target="_blank" rel="noopener noreferrer">
```

Comment:
I think these new links should be `target="_blank"` -- we've made all of the links in open in new windows, since Hubs feels more like an application than a webpage, so user expectation is likely that they will not lose their tab/session/etc by clicking these links to 3rd party sites.

Reply:
gotcha, that now makes more sense.

in that case, can we check for `navigator.standalone`? especially on desktop and mobile Safari, it can be quite the hassle since Safari jumps to the newly opened tab (from `_blank` or `window.open`); Chrome on mobile opens tabs in the background luckily.

to reduce the chance for error, what do you think of a lightweight React component for these external links that adds `rel="noopener noreferrer"` for a link.

and instead of `target="_blank"`, perhaps have a **click** event listener on `document.body`; check if the origin is external, and then open `window.open`.

DRY and reduces possible dev error. thoughts?

Reply:
also would simplify the markup needed in the locale definitions

Reply:
sorry I missed this, I didn't know about `navigator.standalone` -- i think this seems like something to explore in general, probably worth a separate PR at some point that just changes this everywhere if we think it's a good idea